admin/RequestSolrCloudBackups

```diff
@@ -0,0 +1,305 @@
+            }
+        );
+        my $solr_request_status = $solr_response->{status}->{state};
+        if ($solr_request_status ne 'notfound')
+        {
+            _error(
+                "A request to SolrCloud with id '%d' is '%s' already: '%s'",
+                $request_id,
+                $solr_request_status,
+                encode_json($solr_response),
+            );
+            $has_error = 1;
+        }
+    }
+    exit 70 if $has_error; # EX_SOFTWARE
+}
+
+sub do_backup_collections
+{
+    foreach my $request_id (sort keys %backup_requests)
+    {
+        my $backup_name = $backup_requests{$request_id}{name};
+        my $collection = $backup_requests{$request_id}{collection};
+
+        _debug (
+            "Sending backup request '%d' of collection '%s' as '%s'.",
+            $request_id,
+            $collection,
+            $backup_name,
+        );
+        my $solr_response = _query_collections_api('BACKUP', {
+                name => $backup_name,
+                collection => $collection,
+                location => $BACKUP_LOCATION,
+                async => $request_id,
+            }
+        );
+        if (defined $solr_response->{error})
+        {
+            _error(
+                "Failed to backup request '%s' of collection '%s' as '%s': " .
+                "Error '%s'",
+                $request_id,
+                $collection,
+                $backup_name,
+                $solr_response->{error},
+            );
+            _flush_running_requests_status();
+            exit 70; # EX_SOFTWARE
+        }
+        $backup_requests{$request_id}{status} = $BACKUP_REQUEST_RUNNING;
+    }
+}
+
+sub wait_backup_complete
+{
+    while (
+        grep { $backup_requests{$_}{status} eq $BACKUP_REQUEST_RUNNING }
+        keys %backup_requests
+    )
+    {
+        my $pause = '60s';
+        _debug "Sleeping $pause";
+        sleep $pause;
+        foreach my $request_id (
+            sort
+            grep { $backup_requests{$_}{status} eq $BACKUP_REQUEST_RUNNING }
+            keys %backup_requests
+        )
+        {
+            my $solr_response = _query_collections_api('REQUESTSTATUS', {
+                    requestid => $request_id,
+                }
+            );
+            _debug (
+                "Received: %s",
+                encode_json($solr_response),
+            );
+            if (exists $solr_response->{error}) {
+                _error(
+                    "Failed on backup request '%s' of collection '%s' as '%s': " .
+                    "Error '%s'",
+                    $request_id,
+                    $backup_requests{$request_id}{collection},
+                    $backup_requests{$request_id}{name},
+                    $solr_response->{error},
+                );
+                _flush_running_requests_status();
+                exit 70; # EX_SOFTWARE
+            }
+            elsif ($solr_response->{status}->{state} eq 'completed')
```

Comment:
No because this occurrence is rather defined by SolrCloud Collections API.
See above https://github.com/metabrainz/musicbrainz-server/pull/1407#discussion_r405315278.

Reply:
Ok, I thought you intended to use the exact same constants as there (since they are the same, and the doc comments referenced Solr Cloud)